src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1754,9 +1754,12 @@ protected override void AfterMembersCompletedChecks(BindingDiagnosticBag diagnos
 
                 if (TryGetPossiblyUnsupportedByLanguageInlineArrayElementField() is FieldSymbol elementField)
                 {
+                    bool reported_ERR_InlineArrayUnsupportedElementFieldModifier = false;
```

Comment:
>reported_ERR_InlineArrayUnsupportedElementFieldModifier [](http://example.com/codeflow?start=25&length=55)

Consider naming this `reportInlineArrayNotSupported`, and set to `true` in the case of pointers or restricted types below, so the error can be reported in one location rather than two.

Reply:
> Consider naming this ```reportInlineArrayNotSupported```, and set to ```true``` in the case of pointers or restricted types below, so the error can be reported in one location rather than two.

I am not sure what you have in mind exactly. Could you elaborate, please?